src/main/docs/guide/convertToAdoc.adoc

```diff
@@ -19,3 +19,5 @@ Adoc generator based on Apache Freemarker templates, and you can override full t
 |`micronaut.openapi.adoc.output.filename` | Final `adoc` file name. | Default: the same as openapi spec. filename
 |`micronaut.openapi.adoc.templates.*` | Prefix for custom template names. You can override any of the templates that are used in the default implementation to change only part of the resulting document. |
 |===
+
+TIP: See the guide for https://guides.micronaut.io/latest/micronaut-openapi-adoc.html[Generate API Documentation in Asciidoc with the generated OpenAPI specification of your Micronaut application.] to learn more.
```

Comment:
Asciidoc -> AsciiDoc

Reply:
You can suggest these kind of changes directly from the UI 
![CleanShot 2023-12-12 at 09 25 47@2x](https://github.com/micronaut-projects/micronaut-openapi/assets/864788/91db8c7d-8c03-40b1-b722-c248fad0036b)